keel-schema-generator/src/test/kotlin/com/netflix/spinnaker/keel/schema/GeneratorTests.kt

```diff
@@ -618,43 +639,36 @@ internal class GeneratorTests {
     }
 
     @TestFactory
-    fun subTypeDiscriminatorConst() =
-      mapOf(
-        "string" to StringWrapper::class.java.simpleName,
-        "integer" to IntegerWrapper::class.java.simpleName
-      ).map { (fixedValue, subType) ->
-        dynamicTest("the sub-type discriminator property for $subType is an const with the fixed value") {
-          expectThat(schema.`$defs`[subType])
-            .isA<ObjectSchema>()
-            .get { properties[Wrapper<*>::type.name] }
-            .isA<ConstSchema>()
-            .get { const }
-            .isEqualTo(fixedValue)
-        }
-      }
-
-    @TestFactory
-    fun `the discriminator property in the sub-type is an enum with a single value`() =
+    fun `the discriminator property in the sub-type is a const`() =
       mapOf(
         "string" to StringWrapper::class,
         "integer" to IntegerWrapper::class
       )
-        .map { (discriminatorValue, type) ->
-          dynamicTest("the discriminator property of ${type.simpleName} is an enum with the single value \"$discriminatorValue\"") {
-            expectThat(schema.`$defs`[type.simpleName])
-              .isA<ObjectSchema>()
-              .get { properties[Wrapper<*>::type.name] }
-              .isA<EnumSchema>()
-              .get(EnumSchema::enum)
-              .containsExactly(discriminatorValue)
-          }
+        .flatMap { (discriminatorValue, type) ->
```

Comment:
oops. I had `map` instead of `flatMap` so none of the contained tests were running